app/views/wpcc/your_details/new.html.erb

```diff
@@ -93,8 +99,13 @@
             <p class="details__calculate-intro"><%= t('wpcc.details.calculate.details') %></p>
 
             <div class="form__group-item details__calculate-item">
-              <%= f.radio_button(:contribution_preference, 'minimum', class: 'details__calculate-item-select') %>
-              <%= f.label(:contribution_preference_minimum, t('wpcc.details.options.contribution_preference.minimum'), class: 'details__calculate-item-label') %>
+              <% if disable_minimum_contribution_option? %>
+                <%= f.radio_button(:contribution_preference, 'minimum', disabled: 'disabled', class: 'details__calculate-item-select') %>
+                <%= f.label(:contribution_preference_minimum, t('wpcc.details.options.contribution_preference.minimum'), class: 'details__calculate-item-label') %>
```

Comment:
If the only difference between the if and the else clause is the disabled class, you can use a presenter method for that class